prymer/primer3/primer3.py

```diff
@@ -228,13 +227,23 @@ def __init__(
         self,
         genome_fasta: Path,
         executable: Optional[str] = None,
-        variant_lookup: Optional[VariantLookup] = None,
+        vcfs_or_lookup: Optional[list[Path] | VariantLookup] = None,
+        use_cache: bool = False,
+        min_maf: float = 0.0,
+        include_missing_mafs: bool = False,
```

Comment:
**issue** I think we had landed on keeping these out of the `primer3` signature.

IIRC we decided to use the `VariantLookup` default parameters if the user passed paths, and require the user to use `VariantLookup` directly if they wanted more control over these parameters.
```suggestion
```